include/ck/tensor_operation/gpu/device/impl/device_put_element_impl.hpp

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2018-2023, Advanced Micro Devices, Inc. All rights reserved.
+
+#pragma once
+
+#include <iostream>
+#include <sstream>
+
+#include "ck/tensor_description/tensor_descriptor.hpp"
+#include "ck/tensor_description/tensor_descriptor_helper.hpp"
+#include "ck/tensor_operation/gpu/device/reduction_operator_mapping.hpp"
```

Comment:
Remove the including since it is not needed

Reply:
ed4912f2f6a8d6aba6404ebb46b75ed1b39979cd